src/files-and-videos/files-page/data/api.js

```diff
@@ -24,6 +24,19 @@ export async function getAssets(courseId, page) {
   return camelCaseObject(data);
 }
 
+/**
+ * Fetches the course custom pages for provided course
+ * @param {string} courseId
+ * @returns {Promise<[{}]>}
+ */
+export async function getAssetDetails(courseId, filenames) {
+  const params = new URLSearchParams(filenames.map(filename => ['display_name', filename]));
+  params.append('page_size', 100);
```

Comment:
Hate to be the one to ask, but... what happens if there are more than 100 items in the response?  Or is that just not possible, currently?